Makefile

```diff
@@ -66,7 +66,7 @@ proxy:
 	go build -ldflags "-X main.gitCommit=$(COMMIT)" ./hack/proxy
 
 pyenv${PYTHON_VERSION}:
-	virtualenv --python=/usr/bin/python${PYTHON_VERSION} pyenv${PYTHON_VERSION}
```

Comment:
the reason this is here is that it enables a dev with python 2 and python 3 installed to create a pyenv for each locally.  Is it causing an issue in CI?

Reply:
ideally I'd prefer to find a way to maintain this capability.

Reply:
I guess we can do it by installing manually python to this path.
On the agent, using the default task, python is installed inside a `toolbox` path.
I'll check